app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Nice job on adding devise however the confirmation is necessary to give confirmation token before login.
- [ ] And also I can not either sign-up or login due to duplicated files from `devise` in ` user folder` please remove all unnecessary files.